back/Dockerfile

```diff
@@ -111,6 +111,7 @@ RUN \
  ${ASADMIN_PATH}/asadmin --user admin --passwordfile=/opt/pwdfile set resources.custom-resource.elasticsearch.config.property.awsRegion=${ES_SERVER_AWS_REGION} && \
  ${ASADMIN_PATH}/asadmin --user admin --passwordfile=/opt/pwdfile set resources.custom-resource.elasticsearch.config.property.awsAccessKey=${ES_SERVER_AWS_KEY} && \
  ${ASADMIN_PATH}/asadmin --user admin --passwordfile=/opt/pwdfile set resources.custom-resource.elasticsearch.config.property.awsSecretKey=${ES_SERVER_AWS_SECRET} && \
+${ASADMIN_PATH}/asadmin --user admin --passwordfile=/opt/pwdfile set resources.custom-resource.elasticsearch.config.property.indexPrefix=myprefix && \
  ${ASADMIN_PATH}/asadmin --user admin --passwordfile=/opt/pwdfile set resources.custom-resource.office.config.property.office_home=${OFFICE_HOME} && \
```

Comment:
This should be in a different commit/PR, and an environment var must be used (provided by docker-compose.yml file)